conda/recipes/dask-cuda/meta.yaml

```diff
@@ -41,6 +41,11 @@ requirements:
 test:
   imports:
     - dask_cuda
+  commands:
+    - dask-cuda-worker --help
+    - dask-cuda-config --help
+    - dask cuda worker --help
+    - dask cuda config --help
```

Comment:
Considered setting these CLI tests programmatically, but don't see any obvious way to grab the new CLI commands from the pyproject.toml.

cc @jakirkham in case there's something I'm missing

Reply:
There's some code above for the all `-` ones

The ones with ` `s can't be done programmatically AFAIK